### PR TITLE
Pods 7733

### PR DESCRIPTION
**Pull request by cyeniceri**
https://github.com/hmrc/service-manager-config/pull/4775
https://github.com/hmrc/build-jobs/pull/12116

Please merge these 2 PRs first

### CODE REVIEW

File: test/utils/SchemaValidator2ForTests.scala [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    val subSchema = parentSchema.getRefSchemaNode(refNode)
+    val result = factory.getSchema(subSchema).validate(jsonNode).asScala.toSet
+
+    if (result.isEmpty) {
+      JsSuccess(elementToValidate)
+    } else {
+      JsError()
+    }
+
+    //    val deepValidationCheck = true
```
Comment (brownbonnie) [inline test/utils/SchemaValidator2ForTests.scala:53]: Should this code be commented out?

**Review by brownbonnie — DISMISSED** [review]

**Review by grantrobson — APPROVED** [review]

File: test/utils/SchemaValidator2ForTests.scala [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.networknt.schema.{JsonSchemaFactory, SpecVersion}
+import play.api.libs.json._
+
+import java.io.InputStream
+import scala.jdk.CollectionConverters.CollectionHasAsScala
+
+// https://github.com/networknt/json-schema-validator/pull/625
```
[the remark this answers is not in the document]
Comment (grantrobson) [reply test/utils/SchemaValidator2ForTests.scala:27]: Also, why do we have 2 validators now?

[the remark this answers is not in the document]
Comment (cyeniceri) [reply test/utils/SchemaValidator2ForTests.scala:27]: I had added this to explain why the tests are ignored, and we can enable them when this merged. I'll delete the comment and 2nd validator.

[the remark this answers is not in the document]
Comment (cyeniceri) [reply test/utils/SchemaValidator2ForTests.scala]: removed

File: test/utils/SchemaValidator2ForTests.scala (hunk 1 above)
Comment (cyeniceri) [reply test/utils/SchemaValidator2ForTests.scala]: file is removed

**Review by grantrobson — APPROVED** [review]

**Pull request merged**